smb/smb.go

```diff
@@ -1572,3 +2182,35 @@ func (s *Session) NewSetInfoReq(share string, fileId []byte) (SetInfoReq, error)
 		FileId:        fileId,
 	}, nil
 }
+
+func (s *Session) NewQueryInfoReq(
+	share string,
+	fileId []byte,
+	infoType byte,
+	fileInformationClass byte,
+	additionalInformation uint32,
+	outputBufferLength uint32,
+) (QueryInfoReq, error) {
```

Comment:
Add argument for inputBuffer and flags to better support any future extensions for FILE_GET_EA_INFORMATION